app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,8 @@
+class LikesController < ApplicationController
+  def create
+    @post = Post.find(params[:post_id])
+    @like = @post.likes.new(user: current_user)
+    @like.update_counter if @like.save
```

Comment:
There should be a route in the `config/routes.rb` file for creating like. Please create routes to the likes resource so that the user is able to like a post.